readthedocs/api/v2/views/proxied.py

```diff
@@ -0,0 +1,15 @@
+from readthedocs.core.utils.extend import SettingsOverrideObject
+
+from .footer_views import BaseFooterHTML
+
+
+class BaseProxiedFooterHTML(BaseFooterHTML):
+
+    # DRF has BasicAuthentication and SessionAuthentication as default classes.
+    # We don't support neither in the community site.
+    authentication_classes = []
```

Comment:
Is there a reason we don't want them to be present on the footer? Seems like it doesn't hurt anything to have them.

Reply:
Not strong reason on that I guess, I can add it back (or remove the line actually so it takes the defaults)